lib/rspec/core/test_unit_assertions_adapter.rb

```diff
@@ -6,24 +6,9 @@ module Core
     module TestUnitAssertionsAdapter
       include ::Test::Unit::Assertions
 
-      # If using test/unit from Ruby core with Ruby 1.9+, it includes
-      # MiniTest::Assertions by default. Note the upcasing of 'Test'.
-      #
-      # If the test/unit gem is being loaded, it will not include any minitest
-      # assertions.
-      #
-      # Only if Minitest 5.x is included / loaded do we need to worry about
-      # adding a shim for the new updates. Thus instead of checking on the
-      # RUBY_VERSION we need to check ancestors.
```

Comment:
I'd change this to:
```
      # Only if Minitest 5.x is included / loaded do we need to worry about
      # adding a shim, older versions automatically included their assertion module.
      #
      # NB: MiniTest is 4.x (used to be part of Ruby) and Minitest is the gem >= 5.x.
```

Reply:
Actually both [5.x](https://github.com/seattlerb/minitest/blob/v5.14.2/lib/minitest/unit.rb#L18) and [4.x](https://github.com/seattlerb/minitest/blob/v4.7.3/lib/minitest/unit.rb#L1405) have both `Minitest` and `MiniTest` defined:
```ruby
Minitest = MiniTest # :nodoc: because ugh... I typo this all the time
```
```
unless defined?(Minitest) then
  # all of this crap is just to avoid circular requires and is only
  # needed if a user requires "minitest/unit" directly instead of
  # "minitest/autorun", so we also warn
  ...
  MiniTest = Minitest
  ...
end

MiniTest = Minitest unless defined?(MiniTest)
```

Do you mind to only keep the first paragraph?
```
      # Only if Minitest 5.x is included / loaded do we need to worry about
      # adding a shim, older versions automatically included their assertion module.
```